### ISSUE TITLE
No action on save: main.js:52 WebSocket is already in CLOSING or CLOSED state.

### ISSUE THREAD

**Issue by KeyboardRage** [xpyxel/darkbox#8]
**Describe the bug**
Clicking "save" does nothing.

**To Reproduce**
1. Change language to "JavaScript"
2. Write content
3. Click "Save"

**Expected behavior**
Saved, and generate an URL

**Screenshots**
Console is the only indication of anything
https://i.thevirt.us/04/12G09.png

**Desktop (please complete the following information):**
 - OS: Windows 10 OS Build 18363.720
 - Browser: Vivaldi
 - Version 2.4.1488.38 (Stable channel) (32-bit)
- User agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.105 Safari/537.36 Vivaldi/2.4.1488.38

**Comment by xpyxel** [comment]
This should be fixed as of 0d756c15923f9a9277f3907544f2d88d623e74c1, due to removing the websocket code completely and using Ajax instead. It'll be merged into master once v4 is finished.
